package.json

```diff
@@ -81,6 +81,6 @@
 		"trailingComma": "all"
 	},
 	"volta": {
-		"node": "20.x"
+		"node": "24.11.1"
```

Comment:
Node's test runner CLI changed a bit between 20, 22, and 24 (IIRC). Unless there's a need to change this it's probably better left as-is for the moment